en/CommandLine.md

```diff
@@ -1,77 +1,174 @@
 
-Although JabRef is primarily a GUI based application, it offers several command line options that may be useful, and can even perform file conversion operations without opening the graphical interface. The command line options are only available when using the JAR version of JabRef. You can directly download the JAR file from our download site, or, if you already have installed JabRef, you can find the file in your installation folder.
+## Purpose and functioning
 
-You can always specify one or more BibTeX files to load by simply listing their filenames. Take care to specify all options before your list of file names. You must make sure the first file name is not misunderstood as being an argument for an option - this simply means that if a boolean option like `-n` or `-l` immediately precedes a file name, add the word "true" as an argument. For instance, the command line:
+Although JabRef is primarily a GUI based application, it offers several command line options that may be useful, and can even perform file conversion operations without opening the graphical interface.
+The command line options are only available when using the JAR version of JabRef.
+You can directly download the JAR file from our [download site](https://www.fosshub.com/JabRef.html), or, if you already have installed JabRef, you can find the file in your installation folder.
 
-`java -jar JabRef.jar -o filetoexport.xml,docbook -n true     original.bib`
 
-will correctly load the file `original.bib`, export it in docbook format to `filetoexport.xml`, and suppress the GUI. The word *true* prevents the file name from being interpreted as an argument to the -n option.
+## Basics
 
-## Help: -h
+java -jar JabRef.jar [OPTIONS] [BIBTEX_FILE]
 
-This option causes JabRef to display a summary of the command line options, and quit immediately.
+*Note:* you may have to replace `JabRef.jar` by the actual name of the `.jar` file. 
 
-## No-GUI mode: -n
+You can always specify one or more BibTeX files to load by simply listing their filenames.
+Take care to specify all options before your list of file names.
+Ensure that the first file name is not misunderstood as being an argument for an option; this simply means that if a boolean option like `-n` or `-l` immediately precedes a file name, add the word `true` as an argument.
+For instance, the command line:
 
-This option suppresses both the JabRef window and the splash window normally shown at startup. It causes the program to quit immediately once the command line options have been processed.
+`java -jar JabRef.jar -o filetoexport.xml,docbook -n true original.bib`
 
+will correctly load the file `original.bib`, export it in docbook format to `filetoexport.xml`, and suppress the GUI.
+The word *true* prevents the file name from being interpreted as an argument to the `-n` option.
+
+
+## Options
+
+`[-a <FILE>] [-asfl] [-b] [-d <FILE>] [--debug] [-f <FILE>] [-g]
+       [-h] [-i <FILE>] [--importToOpen <FILE>] [-m <FILE>] [-n] [-o
```

Comment:
Based on this there is no argument to the `-n` option. I have no idea how it actually works though.